src/app/components/edit-resource/edit-resource.component.html

```diff
@@ -1,131 +1,150 @@
 <div class="modal-body">
-  <div class="row">
-    <div class="col">
-      <label>Name</label>
-    </div>
-    <div class="col">
+  <div class="row mb-2">
+    <div class="col-6">
+      <label for="name">Name</label>
       <input
+        id="name"
         class="form-control"
         placeholder="Name"
         [(ngModel)]="resource.name"
       />
     </div>
-  </div>
-
-  <div class="row">
-    <div class="col">
-      <label>Abbreviation</label>
-    </div>
-    <div class="col">
+    <div class="col-3">
+      <label for="abbrev">Abbreviation</label>
       <input
+        id="abbrev"
         class="form-control"
         placeholder="Abbreviation"
         [(ngModel)]="resource.abbreviation"
       />
     </div>
-  </div>
-
-  <div class="row">
-    <div class="col">
+    <div class="col-3">
       <label>System</label>
-    </div>
-    <div class="col">
-      <select class="form-control" [(ngModel)]="resource.system">
+      <select class="custom-select" [(ngModel)]="resource.system">
         <option *ngFor="let system of systems" [ngValue]="system">
           {{ system.name }}
         </option>
       </select>
     </div>
   </div>
 
-  <div class="row">
+  <div class="row mb-2">
     <div class="col">
       <label>Type</label>
-    </div>
-    <div class="col">
-      <select class="form-control" [(ngModel)]="resource.resourceType">
+      <select class="custom-select" [(ngModel)]="resource.resourceType">
         <option *ngFor="let type of resourceTypes" [ngValue]="type">
           {{ type.name }}
         </option>
       </select>
     </div>
-  </div>
-
-  <div class="row">
-    <div class="col">
-      <label>OneSky Project ID</label>
-    </div>
     <div class="col">
-      <input
-        class="form-control"
-        placeholder="OneSky Project ID"
-        [(ngModel)]="resource.oneskyProjectId"
-      />
-    </div>
-  </div>
-
-  <div class="row">
-    <div class="col">
-      <label>Description</label>
+      <label>Category</label>
+      <select class="custom-select" [(ngModel)]="resource['attr-category']">
+        <option value="articles">Articles</option>
+        <option value="growth">Christian Growth</option>
+        <option value="conversation_starter">Conversation Starter</option>
+        <option value="gospel">Gospel Invitation</option>
+        <option value="training">Training</option>
+      </select>
     </div>
     <div class="col">
+      <label>Order</label>
       <input
         class="form-control"
-        placeholder="Description"
-        [(ngModel)]="resource.description"
+        placeholder="Order"
+        [(ngModel)]="resource['attr-default-order']"
       />
     </div>
-  </div>
-
-  <div class="row">
-    <div class="col">
-      <label>Youtube Video Id</label>
-    </div>
     <div class="col">
+      <label>OneSky Project ID</label>
       <input
         class="form-control"
```

Comment:
Do you think we need to add the ```for=""``` and ```id=""``` on the label and input for other inputs like you did above?

Reply:
the `for=""` and `id=""` attributes allow you to click on the label to focus the input field. It's not absolutely necessary, especially since this UI is only used by a few internal people. But I can go back and add them on to most of the fields in this form.

Reply:
Okay, I figured it wasn't necessary but didn't know if we wanted to add it to other inputs for consistency.

Reply:
Aside from the checkbox (which I would argue is more necessary than other input types), I added it to one text input but decided it wasn't worth the effort originally ;)